tests/interface/test_cli.py

```diff
@@ -33,15 +33,24 @@ def model_spec(base_config, tmp_path) -> str:
     return filepath
 
 
-@pytest.mark.parametrize("cli_args", [(), ("-i",), ("-l",), ("-i", "-l")])
-def test_simulate_run(runner, model_spec, hdf_file_path, cli_args):
+def test_simulate_run(runner, model_spec, hdf_file_path):
+
+    run_parameters = {param.name for param in simulate.commands["run"].params}
+    expected_parameters = {
+        "model_specification",
+        "artifact_path",
+        "results_directory",
+        "verbose",
+        "quiet",
+        "with_debugger",
+    }
+    different_params = run_parameters.symmetric_difference(expected_parameters)
+    if run_parameters.symmetric_difference(expected_parameters):
```

Comment:
The idea here is to raise in the event we ever change `simulate run` options or args again and forget to update the `expected_parameters`. It should help force us to consider adding an assertion like we do below for `-i` (which is really the only one we need to test since `-o` is inherently tested since we load from there and `-v`, `-q`, and `--pdb` don't really need testing)